docs/specification/1.0.md

```diff
@@ -625,15 +625,15 @@ Note that each hook (e.g. `medication-prescribe`) represents something the user
+Prefetch data, on the other hand, is defined by CDS Services as a way to allow the EHR to provide the data that a CDS Service needs as part of the initial request to the service. When the context data relates to a FHIR resource, it is important not to conflate context and prefetch. For instance, imagine a hook for opening a patient's chart. This hook should include the id of the patient whose chart is being opened, not the full patient FHIR resource. In this case, the FHIR identifier of the patient is appropriate as CDS Services may not be interested in details about the patient resource but instead other data related to this patient. Or, a CDS Service may only need the full patient resource in certain scenarios. Therefore, including the full patient resource in context would be unnecessary. For CDS Services that want the full patient resource, they can request it to be prefetched or fetch it as needed from the FHIR server.
 
 Consider another hook for when a new patient is being registered. In this case, it would likely be appropriate for the context to contain the full FHIR resource for the patient being registered as the patient may not be yet recorded in the EHR (and thus not available from the FHIR server) and CDS Services using this hook would predominantly be interested in the details of the patient being registered.
 
 Additionally, consider a PGX CDS Service and a Zika screening CDS Service, each of which is subscribed to the same hook. The context data specified by their shared hook should contain data relevant to both CDS Services; however, each service will have other specific data needs that will necessitate disparate prefetch requests. For instance, the PGX CDS Service likely is interested in genomics data whereas the Zika screening CDS Service will want Observations.
 
-In summary, context is data specific to a hook and universally relevant to all CDS Services subscribed to said hook. Prefetch data is unique to individual CDS Services and supplements the data from context.
+In summary, context is defined by the hook and typically relevant to all CDS Services subscribed to said hook. Prefetch data is defined by each CDS Service to support the specific behavior provided by the service.
```

Comment:
Thanks - this is better, my brain still struggles a bit with these two concepts. the use of word 'typically relevant' throws me off a bit.  Isn't SHALL in a CDS hooks definition context required?

Is this correct: Context is standardized and published with the hook definition. Prefetch is specific to the vendor providing the CDS service and is not standardized.?

Reply:
How about "In summary, context is defined by the hook because it is dictated by the information available at the point in the workflow at which the hook is triggered. Prefetch data is defined by each CDS Service because it is specific to the information that service needs in order to process."

I'd say your statement is correct, except that I wouldn't say "standardized", I'd just say "defined by".

Reply:
Can we say 'In summary, context is specified in the hook definition to guide developers on the information available at the point in the workflow when the hook is triggered. Prefetch data is defined by each CDS Service because it is specific to the information that service needs in order to process